client/QuickStart/Utils.cs

```diff
@@ -194,43 +95,27 @@ public static KustoConnectionStringBuilder CreateAppCertificateConnectionString(
                 ErrorHandler($"Missing the following required fields in configuration file in order to authenticate using a certificate: {string.Join(", ", missing.Select(item => item.name))}");
 
             var appId = Environment.GetEnvironmentVariable("APP_ID");
-            var SubjectDistinguishedName = Environment.GetEnvironmentVariable("SUBJECT_DISTINGUISHED_NAME");
-            var IssuerDistinguishedName = Environment.GetEnvironmentVariable("ISSUER_DISTINGUISHED_NAME");
-            var privateKeyPemFilePath = Environment.GetEnvironmentVariable("PRIVATE_KEY_PEM_FILE_PATH");
+            var subjectDistinguishedName = Environment.GetEnvironmentVariable("SUBJECT_DISTINGUISHED_NAME");
```

Comment:
Since this is batched together, it's worth saying which variables affect which flow, e.g. by introducing another condition for subject name even though programmatically it's not needed